app/services/app/app.ts

```diff
@@ -84,6 +84,8 @@ export class AppService extends StatefulService<IAppState> {
 
   readonly pid = require('process').pid;
 
+  onboarded: boolean;
```

Comment:
All public properties should be a part of the IAppState to be accessible from all windows